lib/task.js

```diff
@@ -43,7 +44,7 @@ var Task = module.exports = function (opts) {
   }
 };
 
-Task.prototype.ACTIONS = ["help", "run", "concurrent", "install"];
+Task.prototype.ACTIONS = ["help", "run", "concurrent", "envs"];
```

Comment:
This includes the clean up of removing `install` (the underlying Task was remove previously).